docs/src/languages/php/redis.md

```diff
@@ -6,103 +6,49 @@ weight: 7
 
 [Redis](../../add-services/redis.md) is a popular structured key-value service, supported by Platform.sh.  It's frequently used for caching.
 
+## Install PhpRedis
+
 The [PhpRedis](https://github.com/phpredis/phpredis) extension is available on Platform.sh's PHP container images.  However, the extension has been known to break its API between versions when removing deprecated functionality.  The version available on each application image is the latest available at the time that PHP version was built, which if your application is very sensitive to PhpRedis versions may not be ideal.
 
 If the version of the PhpRedis extension available for your PHP version is not compatible with your application and upgrading your application is not feasible, you can use the script below as an alternative to download and compile a precise version of the extension on the fly.
```

Comment:
:hammer_and_wrench: The script is no longer below, so it makes sense to me to change the wording.

```suggestion
If the version of the PhpRedis extension available for your PHP version is not compatible with your application
and upgrading your application is not feasible,
you can use the script linked below as an alternative to download and compile a precise version of the extension.
```